src/components/GuestCard.js

```diff
@@ -1,13 +1,17 @@
 import styled from "styled-components";
 import { useNavigate } from "react-router-dom";
-import { useContext } from "react";
+import { useContext, useState } from "react";
 import { UserContext } from "../util/UserContext";
 import emptyCircle from "../assets/icons/circle_empty.svg";
 import checkedCircle from "../assets/icons/circle-check.svg";
 import detailIcon from "../assets/icons/more-menu-vertical.svg";
+import editIcon from "../assets/icons/edit-line.svg";
+import deleteIcon from "../assets/icons/delete-icon.svg";
 
 export default function GuestCard({ personalData }) {
-  const { name, id, selected } = personalData;
+  const { name, intolerances, notes, id, selected } = personalData;
+  const { deleteGuest } = useContext(UserContext);
+  const [showGuestInfo, toggleShowGuestInfo] = useState(false);
```

Comment:
the setter function should be named setShowGuestInfo. By convention a toggle function toggles a state and does not need to consume a parameter.